lib/storage/file.js

```diff
@@ -932,22 +932,29 @@ File.prototype.getMetadata = function(callback) {
  * @param {object} callback.policy - The document policy.
  *
  * @example
- * file.getSignedPolicy({
+ * var TWO_WEEKS_FROM_NOW = Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14);
```

Comment:
My bad, this doesn't work. Maybe we should accept any input type for `expires`. If it's a number, we'll treat it as seconds since epoch. If it's anything else, we can wrap it in a Date object and call toISOString() on that.

That means this will all work:

```js
options.expires = moment().add(2, 'weeks');

options.expires = Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14);
options.expires = new Date(Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14));

options.expires = new Date('2015-09-13')
options.expires = '2015-09-13';

options.expires = '2015-09-13T15:57:33.092Z';
```